### PR TITLE
SPCTR-2882 -[Buttons] Color background getting transparent after installing the woocommerce plugin.

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
[Buttons] Color background getting transparent after installing the woocommerce plugin.

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

**Review by rahulvarma722 — APPROVED** [review]

**Review by aapatre — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -166,6 +166,8 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Buttons - Color background getting transparent after installing the woocommerce plugin.
```
Comment (bsf-zanev) [inline README.md:170]:
```suggestion
* Fix: Buttons - Resolved an issue in which the background color would appear transparent after installing the WooCommerce plugin.
```

Comment (supriyajjadhav) [reply README.md]: Updated suggested changelog. Thank you!

**Review by AbhijitNage123 — APPROVED** [review]

**Comment by adi3890** [comment]
@AbhijitNage123 can you please check for deprecated file version.
and resolve the merge conflict.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**